client/src/components/PastaReviewForm.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react'
+
+const PastaReviewForm = ({ postReview }) => {
+    const [newReview, setNewReview] = useState({
+        title: '',
+        rating: '',
+        body: '',
+        recipe: ''
+    })
+
+    const handleInputChange = event => {
+        setNewReview({
+            ...newReview,
+            [event.currentTarget.name]: event.currentTarget.value
+        })
+    }
+    
+    const handleSubmit = (event) => {
+        event.preventDefault()
+        postReview(newReview)
+        clearForm()
```

Comment:
What happens if the form is not submitted successfully? Does the form still clear? Would we maybe want to persist the values that the user did supply in the form so that they can correct any errors?